### ISSUE TITLE
Metadata block or easy way to insert document metadata inside text block

### ISSUE THREAD

**Issue by avoinea** [plone/volto#1516]
Have the possibility to add a new block type and/or button in the Text block that takes the content from another field:

**Some use cases are:**

* Display **context/effective** (aka publishing date) anywhere in the content area as a block or within the text block
* Display the **context/subjects** (aka tags) inside the page as a block or within the text block

the above will cover many use cases of creating dynamic pages.

The solution as a [@mention](https://www.draft-js-plugins.com/plugin/mention) in the Text Block, seems giving most advantages as you can insert metadata content anywhere in the page.

Branch: https://github.com/plone/volto/tree/eea-metadata-mentions

![mentions](https://user-images.githubusercontent.com/323385/82670789-7737a280-9c46-11ea-81c0-a4cb3a636d7a.gif)

**Comment by sneridagh** [comment]
@avoinea yes, I knew about the metadata block, not about the mention one! Very nice indeed.

What I was wondering is if it makes sense to add them to core or not.

**Comment by stevepiercy** [comment]
@sneridagh how else would an editor manage meta data on a per page basis, or overriding defaults, if not by adding it to core? I'm not sure why we wouldn't add the feature for the SEO monkeys.

**Comment by avoinea** [comment]
@sneridagh Yes, I think that would be a good addition to the core.
